WebAPI/Controllers/CameraMetadaController.cs

```diff
@@ -37,8 +37,8 @@ public IActionResult Get()
             return BadRequest(result.Message);
         }
 
-        [HttpGet("{id}/GetById")]
-        public IActionResult GetById(int id)
+        [HttpGet("{id}/get_metadata")]
```

Comment:
Use resources/nouns instead of methods: 
reference: 
https://medium.com/@mwaysolutions/10-best-practices-for-better-restful-api-cbe81b06f291